network/p2p/tracer/gossipSubMeshTracer.go

```diff
@@ -196,6 +205,23 @@ func (t *GossipSubMeshTracer) SendRPC(rpc *pubsub.RPC, _ peer.ID) {
 	}
 }
 
+// DuplicateMessage is invoked when a duplicate message is dropped.
+func (t *GossipSubMeshTracer) DuplicateMessage(msg *pubsub.Message) {
+	peerID := msg.ReceivedFrom
+	count, err := t.duplicateMessageTrackerCache.Inc(msg.ReceivedFrom)
+	if err != nil {
+		t.logger.Err(err).
```

Comment:
shouldn't this error be irrecoverable?

Reply:
https://github.com/onflow/flow-go/pull/5058/commits/1b8cf130d20cb44b9f55f3b28afb50f1a1a45e14